serverless/src/index.ts

```diff
@@ -37,11 +38,8 @@ export const handler = async (event: InputEvent, _: any): Promise<OutputEvent> =
       return lambdaOutput;
     }
 
-    return {
-      requestId: event.requestId,
-      status: SUCCESS,
-      fragment,
-    };
+    const stepFunctionOutput = await instrumentStateMachines(event);
```

Comment:
Making them two separate lines so that:
1. it's more clear what `instrumentStateMachines()` returns
2. developers can add `console.log(stepFunctionOutput)` more easily